### PR TITLE
Fix sanity test

### PR DESCRIPTION
**Pull request by aleksvagachev**
##### SUMMARY
Fix sanity test in [postgresql_pg_hba.py](https://github.com/ansible-collections/community.postgresql/blob/main/plugins/modules/postgresql_pg_hba.py)

### CODE REVIEW

File: plugins/modules/postgresql_pg_hba.py [hunk 1]
```diff
@@ -885,6 +884,7 @@ def main():
         if not module.check_mode:
             ret['msgs'].append('Writing')
             try:
+                backup_file = module.params['backup_file']
                 if pg_hba.write(backup_file):
```
Comment (Andersson007) [inline plugins/modules/postgresql_pg_hba.py:888]:
```suggestion
                if pg_hba.write(module.params['backup_file']):
```
a little bit shorter:)

**Review by Andersson007 — COMMENTED** [review]
@aleksvagachev thanks!
Should we add a `bugfixes:` changelog fragment? it looks like a bug imo.

**Comment by aleksvagachev** [comment]
@Andersson007
In my opinion, it is not worth adding. Since the functionality of the module has not been changed, we just corrected sanity/linter test.

**Comment by Andersson007** [comment]
> @Andersson007 In my opinion, it is not worth adding. Since the functionality of the module has not been changed, we just corrected sanity/linter test.

SGTM

**Review by Andersson007 — APPROVED** [review]

**Review by hunleyd — APPROVED** [review]

**Review by jchancojr — APPROVED** [review]

**Comment by Andersson007** [comment]
Folks, I'll merge it but don't hesitate to do it yourself in future:)
- I think two approvals in total including yours is enough in average case
- For typo fixes, feel free to merge them on the spot with only your approval

**Pull request merged**